packages/frontend/src/pages/public/LandingPage.tsx

```diff
@@ -0,0 +1,27 @@
+import firebase from 'firebase/compat/app';
```

Comment:
seems like an unused import? can you double check? @wshe874

Reply:
It might just be a side effect import (in which case it should just be `import "firebase/compat/app"`)

Reply:
do we have a linter setup? eslint or something to make sure that this doesn't happen?

Reply:
> seems like an unused import? can you double check? @wshe874

Thanks for pointing out. It is unused. I will delete it. But yeah, we only have prettier set up.